packages/common/src/deserialization.ts

```diff
@@ -202,3 +202,35 @@ export function deserializeTransaction(
             throw new Error('Invalid blockItemKind');
     }
 }
+
+/**
+ * Deserializes a return value from a sequence of bytes into a json object.
+ * @param returnValueBytes A buffer containing the return value as raw bytes.
+ * @param moduleSchema The raw module schema as a buffer.
+ * @param contractName The name of the contract where the receive function is located.
+ * @param functionName The name of the receive function whose return value you want to deserialize.
+ * @param schemaVersion The schema version as a number. This parameter is optional,
+ * if you provide a serialized versioned schema this argument won't be needed.
+ */
+export function deserializeReturnValue(
+    returnValueBytes: Buffer,
+    moduleSchema: Buffer,
+    contractName: string,
+    functionName: string,
+    schemaVersion?: number
+): any {
```

Comment:
I think that the lint failing is just because of this.
You should add a:
```suggestion
// eslint-disable-next-line @typescript-eslint/no-explicit-any
): any {
```
To appease it. (We normally don't want any types around, but the type here is pretty much any :upside_down_face: )